src/qmapshack/poi/CRawPoi.cpp

```diff
@@ -74,8 +74,87 @@ poi_t CRawPoi::toPoi() const
     poi_t poi;
     poi.pos = coordinates;
     poi.name = getName();
-    poi.desc = data.join("<br>\n");
     poi.icon = garminIcon;
+
+    const QRegularExpression emailRegex ("(contact:email)=(.+)", QRegularExpression::UseUnicodePropertiesOption);
+    const QRegularExpression phoneRegex ("(contact:phone|phone|contact:mobile)=(.+)", QRegularExpression::UseUnicodePropertiesOption);
+    const QRegularExpression websiteRegex ("(contact:website|website|url)=(.+)", QRegularExpression::UseUnicodePropertiesOption);
+    const QRegularExpression osmRegex ("(osm_id)=(P|W|R)/(.+)", QRegularExpression::UseUnicodePropertiesOption);
+    const QRegularExpression wikipediaRegex ("(.*:)?(wikipedia)(:[A-z]{2})?=(.+)", QRegularExpression::UseUnicodePropertiesOption);
+    const QRegularExpression wikidataRegex ("(.*:)?(wikidata)=(.+)", QRegularExpression::UseUnicodePropertiesOption);
+    poi.desc = "";
+    bool firstLine = true;
+    for(QString line : data)
+    {
+        if(firstLine)
+        {
+            firstLine = false;
+        }
+        else
+        {
+            poi.desc += "<br>\n";
+        }
+
+        QRegularExpressionMatch emailMatch = emailRegex.match(line);
+        if(emailMatch.hasMatch())
+        {
+            poi.desc += emailMatch.captured(1).trimmed() + "=<a href=\"mailto:" + emailMatch.captured(2).trimmed() + "\">" + emailMatch.captured(2).trimmed() + "</a>";
```

Comment:
if you take leading and trailing white space out of the regex capture you wouldn't need `trimmed()`.